src/current/v24.1/plpgsql.md

```diff
@@ -409,6 +478,111 @@ BEGIN
   END
 ~~~
 
+### Perform transaction control
+
+Use a `COMMIT` or `ROLLBACK` statement to issue a group of PL/pgSQL statements as an explicit transaction within a [stored procedure]({% link {{ page.version.version }}/stored-procedures.md %}). Any updates made by those statements are either committed or rolled back, while [PL/pgSQL variables](#declare-a-variable) keep their values. 
+
+~~~ sql
+BEGIN
+	statements
+	[ COMMIT | ROLLBACK ]
+~~~
+
+`COMMIT` and `ROLLBACK` statements within PL/pgSQL blocks have the following requirements:
+
+- They must be used inside a PL/pgSQL [stored procedure]({% link {{ page.version.version }}/stored-procedures.md %}).
+{% comment %}- The procedure must be called directly in a `CALL` statement, or all of its ancestors must be stored procedures or `DO` blocks{% endcomment %}.
+- The procedure must be called directly in a [`CALL`]({% link {{ page.version.version }}/call.md %}) SQL statement. It cannot be [called by another stored procedure](#call-a-procedure).
+- There cannot be a [`RAISE EXCEPTION`](#report-messages-and-handle-exceptions) or [`EXCEPTION`](#write-exception-logic) statement preceding the `COMMIT` or `ROLLBACK` statement.
```

Comment:
Actually, RAISE EXCEPTION is allowed. The restriction is specifically that `COMMIT/ROLLBACK` cannot be inside a block that has an exception handler. So that means a procedure like [this one](https://github.com/cockroachdb/cockroach/blob/d15210e0e378c82106543a1b9fb3052ce053cebb/pkg/ccl/logictestccl/testdata/logic_test/plpgsql_txn#L331-L349) is allowed, since the nested block with the exception handler doesn't contain any transaction control statements.

[nit] I'd call it an `EXCEPTION` block or clause, rather than a statement.